pyproject.toml

```diff
@@ -1,5 +1,116 @@
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "cubids"
+description = "Curation of BIDS (CuBIDS): A sanity-preserving software package for processing BIDS datasets."
+authors = [{name = "Neuroinformatics Team of PennLINC"}]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Natural Language :: English",
+    "Topic :: Scientific/Engineering :: Image Recognition",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+license = {file = "LICENSE"}
+requires-python = ">=3.8"
+dependencies = [
+    "pyyaml",
+    "pybids",
+    "pandas",
+    "tqdm",
+    "numpy",
+    "scikit-learn",
+    "datalad>=0.13.5,!=0.17.3,!=0.17.0,!=0.16.1",
+    "wrapt<2,>=1.10",
+    "Sphinx",
+    "jinja2 < 3.1",
+]
+dynamic = ["version"]
+
+[project.urls]
+Homepage = "https://github.com/PennLINC/cubids"
+Documentation = "https://www.cubids.readthedocs.io"
+Paper = "https://doi.org/10.1016/j.neuroimage.2022.119609"
+"Docker Images" = "https://hub.docker.com/r/pennlinc/cubids/tags/"
+
+[project.optional-dependencies]
+doc = [
+    "nbsphinx",
+    "packaging",
+    "sphinx >= 2.2",
+    "sphinx-argparse",
+    "sphinx_rtd_theme",
+    "sphinx_markdown_tables",
+    "recommonmark",
+]
+tests = [
+    "codespell",
+    "coverage",
+    "flake8",
+    "flake8-absolute-import",
+    "flake8-black",
+    "flake8-docstrings",
+    "flake8-isort",
+    "flake8-pyproject",
+    "flake8-unused-arguments",
+    "flake8-use-fstring",
+    "flake8-warnings",
+    "pep8-naming",
+    "pytest",
+    "pytest-cov",
+    "tomli",
+]
+maint = [
+    "fuzzywuzzy",
+    "python-Levenshtein",
+]
+
+# Aliases
+all = ["cubids[doc,maint,tests]"]
+
+[project.scripts]
+cubids-group = "cubids.cli:cubids_group"
+cubids-apply = "cubids.cli:cubids_apply"
+cubids-purge = "cubids.cli:cubids_purge"
+cubids-add-nifti-info = "cubids.cli:cubids_add_nifti_info"
+cubids-copy-exemplars = "cubids.cli:cubids_copy_exemplars"
+cubids-undo = "cubids.cli:cubids_undo"
+bids-sidecar-merge = "cubids.cli:bids_sidecar_merge"
+cubids-validate = "cubids.cli:cubids_validate"
+cubids-datalad-save = "cubids.cli:cubids_datalad_save"
+cubids-print-metadata-fields = "cubids.cli:cubids_print_metadata_fields"
+cubids-remove-metadata-fields = "cubids.cli:cubids_remove_metadata_fields"
+
+#
+# Hatch configurations
+#
+
+[tool.hatch.metadata]
+allow-direct-references = true
+
+[tool.hatch.build.targets.sdist]
+exclude = [".git_archival.txt"]  # No longer needed in sdist
+
+[tool.hatch.build.targets.wheel]
+packages = ["cubids"]
+exclude = [
+    "cubids/tests/data",  # Large test data directory
+]
+
+##  The following two sections configure setuptools_scm in the hatch way
+
+[tool.hatch.version]
```

Comment:
Do you know why github is rendering this in red?

Reply:
No clue. It's doing it in the ASLPrep pyproject.toml as well though...

Reply:
I think it might be a bug on GitHub's end, since installation seems to work just fine.